app/mailers/user_cleanup_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class UserCleanupMailer < ApplicationMailer
+  def cleanup_email(will_remove_users, removed_users)
+    @will_remove_users = will_remove_users
+    @removed_users = removed_users
+    subject = '[Belangrijk!] Er gaan gebruikers worden verwijderd van de website'
```

Comment:
```suggestion
    subject = '[Belangrijk!] Er zijn archiveerbare accounts gevonden op de website'
```

Ofzo. Geeft iets meer aan wat **nu** de status is

Reply:
Ja maar het geeft niet aan dat er iets potentieel gevaarlijks gaat gebeuren. Aangeven dat er dingen verwijderd gaan worden schudt bestuurders wat meer wakker denk ik, dat er wat gevonden is geeft niet aan dat er automatisch wat gaat gebeuren

Reply:
Maar wij zijn de gebruikers, dus hoe belangrijk is wakker schudden? :P

Overigens is dan "x Alpha accounts staan op het punt gearchiveerd te worden" misschien geschikter?

Reply:
Mijn idee was dat het dus danig duidelijk is dat het bestuur actie gaat ondernemen, niet de ICT. "x Alpha accounts staan op het punt gearchiveerd te worden" lijkt me dan inderdaad goed

Reply:
Ah, bestuurders. Jazeker. 👍